### PR TITLE
Update Code to 1.93.0

### PR DESCRIPTION
**Pull request by benz0li**
Fixes https://github.com/coder/code-server/issues/6955

### CODE REVIEW

File: patches/webview.diff [hunk 1]
```diff
@@ -54,21 +54,20 @@ Index: code-server/lib/vscode/src/vs/server/node/webClientServer.ts
...
  			settingsSyncOptions: !this._environmentService.isBuilt && this._environmentService.args['enable-sync'] ? { enabled: true } : undefined,
-@@ -373,7 +374,8 @@ export class WebClientServer {
+@@ -386,7 +387,7 @@ export class WebClientServer {
  			return void res.end('Not found');
  		}
  
--		const webWorkerExtensionHostIframeScriptSHA = 'sha256-V28GQnL3aYxbwgpV3yW1oJ+VKKe/PBSzWntNyH8zVXA=';
-+		const webWorkerExtensionHostIframeScriptSHA = 'sha256-TMJESLUenkUgTCHw3qQlCngteoHhhRnn81kYP41UnCE=';
-+
+-		const webWorkerExtensionHostIframeScriptSHA = isESM ? 'sha256-2Q+j4hfT09+1+imS46J2YlkCtHWQt0/BE79PXjJ0ZJ8=' : 'sha256-V28GQnL3aYxbwgpV3yW1oJ+VKKe/PBSzWntNyH8zVXA=';
```
Comment (benz0li) [inline patches/webview.diff:69]: @code-asher `yarn build:vscode` states `Setting build to ESM: false`.  
👉 No updated hash for ESM (ECMAScript modules).

**Comment by benz0li** [comment]
Ping @code-asher: `code-server-4.93.0-rc.1-linux-amd64.tar.gz` with Code 1.93.0 is deployed at https://coder.jupyter.b-data.ch.

Functionality [modified by patches] tested and found to work:

* base-path
* cli-window-open
* local-storage
* marketplace
* proxy-url
* service-worker
* web view

Workspaces and Jupyter Notebooks also work fine:

*  `ms-toolsai.jupyter@2024.8.0`
*  `ms-python.python@2024.14.0`

@renkun-ken: R Extension: The first `plot` command does not open the "R Plot" window. The window only opens after the second `plot` command.  
👉 You can reproduce it yourself at https://coder.jupyter.b-data.ch yourself. Log in with your GitHub account.

**Comment by benz0li** [comment]
Ping @randy3k: R Extension: The first `plot` command does not open the "R Plot" window. The window only opens after the second `plot` command.  
👉 You can reproduce it yourself at https://coder.jupyter.b-data.ch yourself. Log in with your GitHub account.

**Comment by benz0li** [comment]
FYI @renkun-ken @randy3k: The browser console states

```
[REditorSupport.r]request to http://127.0.0.1:38103/plots failed, reason: socket hang up [mainThreadExtensionService.ts:78:10]

FetchError: request to http://127.0.0.1:38103/plots failed, reason: socket hang up
	at ClientRequest.<anonymous> (/opt/code-server/lib/vscode/extensions/reditorsupport.r-2.8.4-universal/dist/extension.js:2:1943428)
	at ClientRequest.emit (node:events:519:28)
	at emitErrorEvent (node:_http_client:108:11)
	at Socket.socketOnEnd (node:_http_client:535:5)
	at Socket.emit (node:events:531:35)
	at endReadableNT (node:internal/streams/readable:1696:12)
	at process.processTicksAndRejections (node:internal/process/task_queues:82:21) [mainThreadExtensionService.ts:79:10]
```

ℹ️ This does not happen with previous versions of `code-server`.

**Comment by benz0li** [comment]
@code-asher Please check the failing Firefox upload/download end-to-end tests.

Thank you.

**Comment by benz0li** [comment]
@code-asher Code 1.93.1 is out. Unfortunately, I do not have the time to update this PR.

You are on your own.

**Comment by code-asher** [comment]
I will try to get to it today, but it might be next week...

**Comment by benz0li** [comment]
@code-asher Thanks for all your efforts.

**Comment by code-asher** [comment]
Still not sure what is going on with context menus in Firefox, but I verified the tests manually so for now I am going to disable them and experiment later.  Will merge now and get a release out soon-ish.

**Review by code-asher — APPROVED** [review]

**Pull request merged**